extensions/mysql/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const Promise = require('bluebird');
+const mysql = require('mysql');
```

Comment:
the CLI doesn't depend on `mysql` explictly. Although it's added indirectly by knex-migrator, we need to add this dep to Ghost-CLI itself if we want to ensure things to work right in all installs.